examples/js/hello-world/app.js

```diff
@@ -94,9 +94,9 @@ app.post('/send/submit', (req, res) => {
     const call = req.body;
 
     let message = 'Hello, world!';
-    const user_message = call.values.message;
-    if (user_message) {
-        message += ' ...and ' + user_message + '!';
+    const submitted_message = call.values.message;
+    if (submitted_message) {
+        message += ' ...and ' + submitted_message + '!';
```

Comment:
Any reason why we are using snake_case here? My original request was made because I was suggesting to use camelCase, though I didn't make that clear in the request. The typical JS convention is definitely to use camelCase (especially for variable names), though snake_case is usually used for json payloads being sent over the wire.

Reply:
I _thought_ snake_case was used in the webapp, but that isn't the case actually. I've converted to camelCase.